client-react/src/pages/app/deployment-center/container/DeploymentCenterContainerForm.tsx

```diff
@@ -825,7 +825,7 @@ const DeploymentCenterContainerForm: React.FC<DeploymentCenterContainerFormProps
             />
           </>
           <div className={pivotContent}>
-            <DeploymentCenterContainerPivot {...props} formProps={formProps} />
+            <DeploymentCenterContainerPivot {...props} tab={props.tab} formProps={formProps} />
```

Comment:
`tab` should already be set because of the spread before it.

```suggestion
            <DeploymentCenterContainerPivot {...props} formProps={formProps} />
```